rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,21 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+* Log SHALL be drop in compatible with the standard library logger.
+* Log SHALL use key / value pairs for structured data.
+* Log MAY allow multiple stored logging contexts each with a different set of k/v pairs.
+* Log SHALL be transport agnostic by accepting and io.Writer as output target.
+* Log SHALL provide a default stdout io.Writer.
+* Log MAY implement io.MultiWriters allowing broadcast of logs over multiple transports.
```

Comment:
We (ab)use ioMultiWriter for this, but it's sort of a hack since writes != messages, it works since there's no partial writes going. Using io.Writers is probably not the best interface, it requires that you're fully aware of _which_ io.Writers are being used, special-cased for messages